src/main/kotlin/app/extractors/CExtractor.kt

```diff
@@ -0,0 +1,29 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Liubov Yaronskaya (lyaronskaya@sourcerer.io)
+
+package app.extractors
+
+import app.model.CommitStats
+import app.model.DiffFile
+
+class CExtractor : ExtractorInterface {
+    override fun extract(files: List<DiffFile>): List<CommitStats> {
+        TODO("not implemented")
```

Comment:
This mean that exception will be thrown if this method will be called. Not a good idea to merge such things to master.